### PR TITLE
Add introductory blog post about OES structure and governance

### PR DESCRIPTION
**Pull request by tobie**

### CODE REVIEW

File: content/blog/how-we-work [hunk 1]
```diff
@@ -0,0 +1,31 @@
++++
+title = "How we work"
+heading = "Understanding the Organization for Ethical Source’s structure and governance model"
+summary = "This is the first post of a series explaining the Organization for Ethical Source’s structure, governance model, practices, and norms."
++++
+
+<time>May 3, 2021</time>
+
+As you know, the Ethical Source movement [organized into a legal entity](/blog/oes-announcement/) in late January.
```
Comment (CoralineAda) [inline content/blog/how-we-work]: I would take out the "As you know", and maybe move this sentence to the first paragraph

**Review by CoralineAda — APPROVED** [review]

**Comment by CoralineAda** [comment]
@tobie Check out the deploy preview-- something's wrong with that page. https://deploy-preview-98--ethicalsource.netlify.app/blog/how-we-work

**Review by CoralineAda — CHANGES_REQUESTED** [review]
https://deploy-preview-98--ethicalsource.netlify.app/blog/how-we-work is not rendering, I'll take a look.

**Comment by CoralineAda** [comment]
@tobie Just missing the `.md` extension on that file, then it should be good.

**Comment by tobie** [comment]
> @tobie Just missing the `.md` extension on that file, then it should be good.

🤦‍♂️

**Review by CoralineAda — APPROVED** [review]

**Comment by CoralineAda** [comment]
Looks great!

**Pull request merged**